pages/design/ui-patterns/progressive-disclosure.mdx

```diff
@@ -10,10 +10,10 @@ The following table outlines common progressive disclosure solutions in use at G
 
 | Component | Icon | Usage | Notes |
 | --- | --- | --- | --- |
-| Chevron | ![chevron](https://user-images.githubusercontent.com/24916540/52089727-1ae9d480-2564-11e9-8291-1ee9d3225427.png) | When elements of content are collapsed and can be toggled open | Do not use this icon to trigger a dropdown menus or navigation; the Caret icon is the more suitable for this. | 
-Expand/ Collapse Icon | ![expand](https://user-images.githubusercontent.com/24916540/52089724-1a513e00-2564-11e9-84f8-eba077f6abfc.png) | Signify that there is content (typically text) to be revealed | Should generally stand alone, rather than being paired with text | 
-Ellipses Icon | ![ellipses](https://user-images.githubusercontent.com/24916540/52089725-1ae9d480-2564-11e9-8d29-1dc28740924a.png) | For toggling truncated inline text content | Do not use this icon to trigger a dropdown menus or navigation; the Kebab icon is the more suitable for this. | 
-Text-only Toggles | | | Usage of this solution is discouraged, as generally icons or icon+text pairings provide better accessibility and more information |
+| Chevron | ![chevron](https://user-images.githubusercontent.com/24916540/52089727-1ae9d480-2564-11e9-8291-1ee9d3225427.png) | When elements of content are collapsed and can be toggled open | Do not use this icon to trigger a dropdown menus or navigation; the Caret icon is the more suitable for this. |
+Expand/ Collapse Icon | ![expand](https://user-images.githubusercontent.com/24916540/52089724-1a513e00-2564-11e9-84f8-eba077f6abfc.png) | Signify that there is content (typically text) to be revealed | Should generally stand alone, rather than being paired with text |
+Ellipses Icon | ![ellipses](https://user-images.githubusercontent.com/24916540/52089725-1ae9d480-2564-11e9-8d29-1dc28740924a.png) | For toggling truncated inline text content | Do not use this icon to trigger a dropdown menus or navigation; the Kebab icon is the more suitable for this. |
+Text-only Toggles | [example]() |  | Usage of this solution is discouraged, as generally icons or icon+text pairings provide better accessibility and more information |
```

Comment:
Thanks for adding this. There are a few things I'm wondering about here – 

- _Do_ we generally discourage them? Note [this](https://www.dropbox.com/s/gq5di6wy11fbi62/Screenshot%202019-06-26%2011.31.35.png?dl=0) on the dashboard page
- Should we include text-only toggles if we generally discourage them? If we're aligned that this is *not* a pattern we want to encourage, I don't think we should include this at all in the table. 🤔 
- If we end up keeping text-only toggles in this table, it might make sense to use the text 'Show more' instead of 'example' just to make it a bit more clear

Reply:
Agreed. I meant to ask you where else I could find text toggles. It seems like the only place I can find this is on the dashboard there. If we can find a few more instances, maybe we can get a better idea.

Reply:
Just chatted with @emplums about this – let's remove it from the table unless we think there's a valid reason to keep it